wcfsetup/install/files/lib/system/form/builder/field/acl/AclFormField.class.php

```diff
@@ -192,8 +192,10 @@ public function readValue()
     /**
      * @inheritDoc
      */
-    public function cleanup()
+    public function cleanup(): static
     {
         ACLHandler::getInstance()->resetValues($this->getObjectType()->objectTypeID);
+
+        return $this;
```

Comment:
As this is an actual bug: Can you split this in a separate commit that I can apply independently to 5.5?